crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/rpc.rs

```diff
@@ -71,7 +71,7 @@ impl LedgerRpcProvider for LedgerDB {
             l2_block_ids.len() <= MAX_L2_BLOCK_PER_REQUEST as usize,
             "requested too many l2 blocks. Requested: {}. Max: {}",
             l2_block_ids.len(),
-            MAX_BATCHES_PER_REQUEST
+            MAX_L2_BLOCK_PER_REQUEST
```

Comment:
I noticed this while debugging this PR... we are comparing against `MAX_L2_BLOCK_PER_REQUEST` so that is what we should have in the error message.